### DISCUSSION TITLE
posts/book-review-1/

### DISCUSSION THREAD

**Discussion by giscus** [SunnyCloudYang/SunnyCloudYang.github.io#25] [Comments]
# posts/book-review-1/

理想与物质的倒挂，选择与未来的矛盾，让每个醒来的人在这里挣扎

https://origakid.top/posts/book-review-1/

**Comment by dkn16** [comment]
我这方面比较坚持认为有些人其实并不清楚自己的需求 -- 崇高理想很多时候意味着没有/极少的物质回馈，有的人被裹挟了，其实似乎只是觉得“这样好”或者“这样可以证明我强于他人”而并非发自内心的欣喜。大部分人始终第一向往的是更好的物质生活，这才是正常的。
（最后祝您早日找到真正满意的事业

**Reply by SunnyCloudYang** [reply]
You took my words. 
我觉得追求崇高理想/物质回馈都无可非议，最重要的还是知道**自己想要**什么，而不是**别人想要**自己做什么，或者说应该追求自身评价标准下的成功而不是社会评价体系下的成功。只是这两种生活往往不是完全重合的，也会面临一些权衡，当路径与目标错位时，大概就会被裹挟产生“随大流“的盲目状态。当然对于大多数人来说，更合适的还是首先追求物质生活（可是不少人迫于很多因素并没有如此选择）。
也祝您在探寻宇宙真相之路上收获满满！
